README.md

```diff
@@ -14,10 +14,13 @@ The project operates on an open contribution model, welcoming contributions from
 To see what's incoming, check the open [pull requests](https://github.com/greenelab/deep-review/pulls).
 For project discussion and planning see the [Issues](https://github.com/greenelab/deep-review/issues).
 
-### Current stage: resubmitted to _JRSI_ and _bioRxiv_
+### Current stage: planning Deep Review 2019
 
 We [completed](https://github.com/greenelab/deep-review/releases/tag/v0.10) manuscript revisions in [response](content/response-to-reviewers.md) to two external peer reviews.
-Please see [issue #810](https://github.com/greenelab/deep-review/issues/810) for a discussion about future plans and whether we will continue to accept new pull requests.
+As of writing, we are aiming to publish an update of the deep review each year, with the next such release occuring in June 2019.
+We will continue to make project preprints available on bioRxiv, and aim to continue publishing the finished reviews in a peer-reviewed venue as well.
```

Comment:
As a practical matter, we won't be able to update the manuscript in bioRxiv.  We would have to create a new version as a new entity or use a different preprint server.  Either could be good options.  What you have written is accurate though, I don't suggest changing it.

It is also unknown at this point which publishers would be interested in an updated version.  I suspect it depends heavily on how much the manuscript is updated.